articles/aks/gpu-cluster.md

```diff
@@ -133,6 +133,8 @@ In order to demonstrate the GPUs are indeed working, schedule a GPU enabled work
 
 The following job manifest includes a resource limit of `alpha.kubernetes.io/nvidia-gpu: 1`. The appropriate CUDA libraries and debug tools will be available on the node at `/usr/local/nvidia` and must be mounted into the pod using the appropriate volume specification as seen below.
 
+[!NOTE] Host drivers currently support CUDA 8.0. Using 9.0 and above will most likely throw a version mismatch error when calling into drivers, for example: `CUDA driver version is insufficient for CUDA runtime version`.
+
```

Comment:
Rather than callout specific CUDA versions how about we share the source with the driver version we are installing - eg. https://github.com/Azure/acs-engine/blob/9993e58818f8355be96278c15006e3d6d2be235f/pkg/acsengine/engine.go#L462

Reply:
👍 on referencing driver version from acs-engine. This link might also be helpful for folks looking for more detailed hardware support: https://docs.nvidia.com/deploy/cuda-compatibility/index.html